lib/segment/src/fixtures/index_fixtures.rs

```diff
@@ -68,6 +68,10 @@ impl<TMetric: Metric<VectorElementType>> VectorStorage for TestRawScorerProducer
         self.vectors.len()
     }
 
+    fn available_size_in_bytes(&self) -> usize {
+        self.available_vector_count() * self.vector_dim() * std::mem::size_of::<VectorElementType>()
+    }
```

Comment:
Though I don't see us reaching u64 limits here per se, should we stay on the safe side and use [`saturating_mul`](https://doc.rust-lang.org/std/primitive.usize.html#method.saturating_mul)?

Maybe in the other places we do such calculation too.

Reply:
I guess it's safe here because size in bytes on disk cannot be larger than `u64::max`